File: adhoc/aneduarte/1961.py
# -*- coding: utf-8 -*-

dados = input().split(" ")
P = int(dados[0])
N = int(dados[1])

pulos = input().split(" ")
pulo = int(pulos[0])

for i in range(1, N, 1):
    pulo_seguinte = int(pulos[i])
    if (abs(pulo-pulo_seguinte) > P):
        print("GAME OVER")
        break
    pulo = pulo_seguinte
    if (i == N-1):
        print("YOU WIN")

File: adhoc/lucianoclaudio/1961.cpp
#include <bits/stdc++.h>

using namespace std;

int main(){

    vector<int> canos;
    int i,p,n,x;
    bool pulo;

    cin >> p >> n;

    for (i=0;i<n;i++){
        cin >> x;
        canos.push_back(x);
    }
    for (i=0;i<n-1;i++){
        if(abs(canos[i]-canos[i+1]) <= p){
            pulo=true;
        }
        else{
            pulo=false;
            break;
        }
    }
    if(pulo==true){
        cout << "YOU WIN\n";
    }
    else {
        cout << "GAME OVER\n";
    }

    return 0;
}